.github/workflows/publish.yaml

```diff
@@ -83,11 +83,11 @@ jobs:
         uses: aws-actions/configure-aws-credentials@v4
         with:
           aws-region: us-east-1
-          role-to-assume: arn:aws:iam::403483446840:role/autogen_role_beta_github_actions_release_asana_client_libraries
+          role-to-assume: arn:aws:iam::403483446840:role/autogen_github_actions_beta_release_asana_node_client_library
       - name: Load secrets
```

Comment:
I also ran `z secret add_roles --name npm_api_key --cluster beta github_actions_release_asana_node_client_library` to make sure that the new role has access to the secrets needed!